libs/src/services/ethContracts/governanceClient.js

```diff
@@ -394,6 +395,26 @@ class GovernanceClient extends ContractClient {
       blockNumber: voteEvent.blockNumber
     }
   }
+
+  /**
+   *
+   * @param {Integer} proposalId id of the governance proposal
+   * @returns {BigNumber} amount of tokens required to reach quorum
+   */
+  async calculateQuoroum (proposalId) {
+    const { submissionBlockNumber } = await this.getProposalById(proposalId)
+
+    // represented as a value > 0, eg 5% is 5
+    const quoroumPercent = await this.getVotingQuorumPercent()
+
+    // retrieve stake at the time of proposal from Staking client
+    const totalStakeAtProposal = await this.stakingProxyClient.totalStakedAt(submissionBlockNumber)
+
+    // quorum = (total staked at proposal * quorum percent) / 100
+    const quorumStake = (totalStakeAtProposal.mul(Utils.toBN(quoroumPercent))).div(Utils.toBN(100))
```

Comment:
note: since BN doesn't support decimals, this will be effectively a round down. under the hood, it does divmod and returns only the divisor. Not sure if this matters in practice or not
https://github.com/indutny/bn.js/blob/6ef485dc30b002ec8100602998763fec8a1a3897/lib/bn.js#L2544

Reply:
this is interesting. effectively i don't think it matters because the precision is to the 18th power and we'll drop all of that to get the actual token count i can use divround or pass back the divmod result. 

but what do you think is the best output here? should it just be in wei? i can also return the number in rounded AUDIO

Reply:
I think the best UI is $AUDIO rounded up with hover action that shows full wei (also rounded up). We show decimals in the UI for $AUDIO based on wei amts, so it does matter. In practice, unless someone complains, i'm ok shipping this as long as we put a comment saying this will round down to the nearest wei

Reply:
can you take another look at this, i think i fixed this now. if there's a remainder i round up